source/release-notes.rst

```diff
@@ -15,6 +15,123 @@ Releases
    :depth: 1
 
 
+CIC_00.20.02.24_A09
+======================
+
+* This is a Pre-Production Release for evaluation and development purposes and it cannot be used for production purposes.
+This release is supported on CML NUC - NUC10i7FN* Celadon in Container.
+
+Previous Release : CIC_00.20.02.23_A09
+
+New Features
+-------------
+
+* Enable File based encryption on secure flash.
+* Add HostKeymaster support.
+* Provide Settings screen to set ethernet proxy settings.
+* Fix for Audio is not routing over HDMI and USB headset.
+* Fix for CTS execution completion.
+* Fix for Bluetooth crash issue.
+
+Existing Features
```

Comment:
although Trusty is disabled, SELinux and dm-verity are still there. better to mention them here.

Reply:
Updated PR (y)